test/jasmine/tests/sankey_test.js

```diff
@@ -313,30 +319,23 @@ describe('sankey tests', function() {
         });
 
         it('Plotly.plot shows and removes tooltip on node, link', function(done) {
-
             var gd = createGraphDiv();
             var mockCopy = Lib.extendDeep({}, mock);
 
-            Plotly.plot(gd, mockCopy)
-                .then(function() {
-
-                    mouseEvent('mousemove', 400, 300);
-                    mouseEvent('mouseover', 400, 300);
-
-                    window.setTimeout(function() {
-                        expect(d3.select('.hoverlayer>.hovertext>text').node().innerHTML)
-                            .toEqual('447TWh', 'tooltip present');
-
-                        mouseEvent('mousemove', 450, 300);
-                        mouseEvent('mouseover', 450, 300);
-
-                        window.setTimeout(function() {
-                            expect(d3.select('.hoverlayer>.hovertext>text').node().innerHTML)
-                                .toEqual('46TWh', 'tooltip jumped to link');
```

Comment:
it's there: https://github.com/plotly/plotly.js/pull/1582/files#diff-6c72cf011d0c1bbd8d91d759790188aeR330